src/Raven.Client/Documents/Operations/GetAttachmentOperation.cs

```diff
@@ -135,7 +135,7 @@ public override void SetResponseUncached(HttpResponseMessage response, Stream st
                 var etag = response.GetRequiredEtagHeader();
                 IEnumerable<string> hashVal;
                 var hash = response.Headers.TryGetValues("Content-Hash", out hashVal) ? hashVal.First() : null;
-                var size = response.Content.Headers.ContentLength ?? 0;
+                var size = stream.Length;
```

Comment:
I don't think that we'll always know that what the length is here. 
Doesn't it require us to read the whole thing sometimes?